interface/weno/newSearchRxNorm.php

```diff
@@ -0,0 +1,63 @@
+<form name="druglookup" method="post" action="">
```

Comment:
Need license/author header, and include of globals.php to ensure only authenticated users use this script.